custom_components/vicare/climate.py

```diff
@@ -278,6 +284,16 @@ def set_hvac_mode(self, hvac_mode):
                 f"Cannot set invalid vicare mode: {hvac_mode} / {vicare_mode}"
             )
 
+        supported_modes = self._attributes["vicare_modes"]
+        # Alternative heating modes for systems without DHW
```

Comment:
FYI maybe it's a bit misleading comment, as e.g. my system overall is with DHW :thinking: (but this could still help)

Reply:
I agree. I need to get a better picture of "what modes are available on what kind of systems" before moving on with this "fallback". On the other hand I might just remove the comment and merge it as a first step to improve the current situation for many users

Reply:
Sounds reasonable.